build.xml

```diff
@@ -374,7 +374,7 @@ NOTE at this time jdk8 must be used as there is still a dependency on tools.jar
                 </restrict>
                 <chainedmapper>
                     <flattenmapper/>
-                    <regexpmapper from="^(?!LoboBrowser-1.0.0.jar)(.*)" to="\1"/>
+                    <regexpmapper from="^(?!Cobra-1.0.2|LoboBrowser-1.0.0.jar)(.*)" to="\1"/>
```

Comment:
need  .jar on the Corbra-1.0.2   (that fixed my issue on windows)

Reply:
oh, oops. I must have though that the .jar was outside the grouping. Good find.